kodilla-advanced-tests/src/test/java/com/kodilla/parametrized_tests/homework/GamblingMachineTestSuite.java

```diff
@@ -0,0 +1,67 @@
+package com.kodilla.parametrized_tests.homework;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvFileSource;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+public class GamblingMachineTestSuite {
+
+    @ParameterizedTest
+    @CsvFileSource(resources = "/GoodNumbers.csv")
```

Comment:
Te testy nie testuja w ogole tego, co powinny testowac. 
Plik z danymi tez jest niepoprawny.
Jedyne, co mozesz tu przetestowac, to czy metoda rzuci wyjatek, czy nie.